racksdb/web/ui/src/views/DatacenterDetailsView.vue

```diff
@@ -7,16 +7,41 @@ SPDX-License-Identifier: GPL-3.0-or-later -->
 <script setup lang="ts">
 import { useHttp } from '@/plugins/http'
 import { useRacksDBAPI } from '@/composables/RacksDBAPI'
-import { ref, onMounted, watch } from 'vue'
-import SearchBar from '@/components/SearchBar.vue'
+import { ref, onMounted, watch, computed } from 'vue'
 import type { Ref } from 'vue'
 import type { Datacenter, DatacenterRoom } from '@/composables/RacksDBAPI'
 import BreadCrumbs from '@/components/BreadCrumbs.vue'
+import { BarsArrowDownIcon, BarsArrowUpIcon, ChevronRightIcon } from '@heroicons/vue/24/outline'
+import { useRouter } from 'vue-router'
 
+const router = useRouter()
 const http = useHttp()
 const racksDBAPI = useRacksDBAPI(http)
 const datacenters: Ref<Array<Datacenter>> = ref([])
 const datacenterDetails: Ref<Datacenter | undefined> = ref()
```

Comment:
`datacenterDetails` usage could now be avoided in template, thus making it useless to be a Ref. It is also true for `datacenters` which is used only once in `getDatacenters()`.

Reply:
I removed the Ref from `datacenters`, but i didn't removed it from `datacenterDetails` because this caused a malfunction in `filteredRooms`.